entities/Expense.entity.ts

```diff
@@ -0,0 +1,44 @@
+import { ApiHideProperty, ApiProperty, ApiPropertyOptional } from "@nestjs/swagger";
+import { IsCurrency, IsDefined, IsOptional, IsUUID, Length } from "class-validator";
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from "typeorm";
+import { Auditable } from "./Auditable";
+import { Company } from "./Company.entity";
+
+@Entity('expense')
+export class Expense extends Auditable{
+    @PrimaryGeneratedColumn('uuid')
+    @IsOptional()
```

Comment:
I guess this will not be optional, maybe an option is to remove it @fernandogcabal 

Right @matheo97 ?

Reply:
Done